src/editor/helpers/SpokeParticleHelper.js

```diff
@@ -0,0 +1,53 @@
+import THREE from "../../vendor/three";
+import { addIsHelperFlag } from "./utils";
+
+export default class SpokeParticletHelper extends THREE.Object3D {
```

Comment:
Particlet -> Particle

Reply:
Thanks John! To update, this script is obsolete and has been deleted :))